backend/migrations/versions/4b77ac3913c1_add_last_leaderboard_reward_date_to_.py

```
@@ -0,0 +1,38 @@
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('test_case_outputs', schema=None) as batch_op:
+        batch_op.create_index(batch_op.f('ix_test_case_outputs_test_case_id'), ['test_case_id'], unique=False)
+
+    with op.batch_alter_table('users', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('last_leaderboard_reward_date', sa.Date(), nullable=True))
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
```

Ruff (0.8.2)

Missing return type annotation for public function `downgrade`

Add return type annotation: `None`

(ANN201)